packages/shared/src/components/streak/popup/DayStreak.tsx

```diff
@@ -58,15 +59,25 @@ export function DayStreak({
   const finalDay = day < 7 ? day : day % 7;
 
   return (
-    <div className="relative flex flex-col items-center gap-1">
-      {shouldShowArrow && (
-        <TriangleArrowIcon
-          className="absolute -top-4 text-theme-color-bacon"
-          size={IconSize.XXSmall}
-        />
-      )}
-      {renderIcon()}
-      {!isNullOrUndefined(day) ? dayInitial[finalDay] : null}
-    </div>
+    <WithSimpleTooltip
```

Comment:
I didn't really want to wrap `SimpleTooltip` around the non-weekend circles

Reply:
Hmm and by using disabled? (I know it's still there technically)
If we go this route, can we maybe add the show prop on the simpleTooltip?

Think it will make cleaner/more versatile use for later cases.

Reply:
Good idea, done.